demo/full/scripts/controllers/ContentList.tsx

```diff
@@ -500,19 +517,29 @@ function ContentList({
   );
 
   const onClickLoad = () => {
+    let content: IPlayableContent;
     if (contentChoiceIndex === 0) {
-      const drmInfos = [
-        {
-          licenseServerUrl,
-          serverCertificateUrl,
-          drm: chosenDRMType,
-          customKeySystem,
-        },
-      ];
-      loadUrl(currentManifestURL, drmInfos);
+      content = {
+        url: currentManifestURL,
+        transport: transportType.toLowerCase(),
+        fallbackKeyError: !!shouldFallbackOnKeyError,
+        fallbackLicenseRequest: !!shouldFallbackOnLicenseReqError,
+        isLowLatency: isLowLatencyChecked,
+        drmInfos: [
+          {
+            licenseServerUrl,
+            serverCertificateUrl,
+            drm: chosenDRMType,
+            customKeySystem,
+          },
+        ],
+      };
     } else {
-      loadContent(contentsToSelect[contentChoiceIndex]);
+      content = contentsToSelect[contentChoiceIndex];
     }
+    loadContent(content)
+      // eslint-disable-next-line no-console
+      .catch((err) => console.log(err));
```

Comment:
From what I read, `loadContent` both never rejects but also already has a `console.error` when it throws inside of it.
So the effort of logging + the eslint disabling line is kind of weird when you read it.